cinema/utilities.py

```diff
@@ -7,10 +7,16 @@
 
 
 def send_new_product_notification(product):
-    if settings.ALLOWED_HOSTS:
-        host = 'http://' + settings.ALLOWED_HOSTS[0]
-    else:
-        host = 'http://localhost:8000'
+    """
+    Function, that is called by 'post_save_dispatcher' signal handler.
+
+    Send notification messages to registered users about appearance of new
+    movie on blu-ray.
+    """
+    host = 'http://{0}'.format(
+        settings.ALLOWED_HOSTS[0] if settings.ALLOWED_HOSTS else
+        'localhost:8000'
+    )
```

Comment:
What about the host building into template and use `request` object?
```
{{ request.path }}  #  -without GET parameters 
{{ request.get_full_path }}  # - with GET parameters
```
Is it your case?

Reply:
I can not send request object to context.